docs/tasks/administer-cluster/kubeadm-upgrade-ha.md

```diff
@@ -0,0 +1,179 @@
+---
+reviewers:
+- jamiehannaford 
+- luxas
+- timothysc 
+- jbeda
+title: Upgrading kubeadm HA clusters from 1.8.x to 1.9.x
+---
+
+{% capture overview %}
+
+This guide is for upgrading `kubeadm` HA clusters from version 1.8.x to 1.9.x, as well as 1.8.x to 1.8.y and 1.9.x to 1.9.y where `y > x`. The term "`kubeadm` HA clusters" refers to clusters of more than one master node created with `kubeadm`. For versions 1.8.x and 1.9.x setting up such clusters is not directly supported by `kubeadm` and hence involves some manual steps. See 
+[Creating HA clusters with kubeadm](/docs/setup/independent/high-availability/) for instructions on how to do this. The upgrade procedure described here targets clusters created following those very instructions. You may also want to read [Upgrading/downgrading kubeadm clusters between v1.8 to v1.9](/docs/tasks/administer-cluster/kubeadm-upgrade-1-9/) for more information on upgrading clusters with `kubeadm`.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+Before proceeding:
+
+- You need to have a functional `kubeadm` HA cluster running version 1.8.0 or higher in order to use the process described here. Swap also needs to be disabled.
+- Make sure you read the [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.9.md) carefully.
+- Note that `kubeadm upgrade` will not touch any of your workloads, only Kubernetes-internal components. As a best-practice you should back up what's important to you. For example, any app-level state, such as a database an app might depend on (like MySQL or MongoDB) must be backed up beforehand.
```

Comment:
Recommend wording changes:
- "what's" -> "anything"
- "app" -> "applications"
Typo:
- "an" -> "and"

You say "best practices" and "should" then followed by "must" in the second sentence. I'm totally fine with using must through-out, but it should be consistent - either mandatory or suggested.